app/views/site/id.html.erb

```diff
@@ -23,7 +23,7 @@
       .imagery(iD.data.imagery)
       .taginfo(iD.services.taginfo())
       .embed(true)
-      .assetPath("/iD/") <%# Cant use asset_path('iD/') in production. %>
+      .assetPath("iD/")
       .assetMap(<%= assets("iD").to_json.html_safe %>)
```

Comment:
So, funny thing.  I don't think our current code was using the `assetMap` at all.  I watch what iD does in Chrome dev tools and it fetches all the assets from `/iD/` because of that line.  There is a symlink from `/public/iD` -> `/vendor/assets/iD` which seems kind of hacky.

Going forward I _think_ it will use the assetMap, because `1. we precompile everything and 2. resources in the map don't have the leading slash`.

That comment goes way back from before we even supported the assetMap call and has gone through several revisions in what path it refers to.